### DISCUSSION TITLE
Can you export an existing pdf to json for create feature?

### DISCUSSION THREAD

**Discussion by darrenparkinson** [pdfcpu/pdfcpu#980] [Q&A]
I haven’t played with this yet but wanted to see if it might be something I could use. Essentially I have an existing pdf I’d like to recreate with different data — is this possible? Couldn’t see anything immediately in the docs. Apologies if this is obvious.

**Comment by hhrutter** [comment]
Hello!

Quite complex task at hand..
That use case is so general I doubt there is anything out there for you unless we are talking about a PDF form:

You can check for form fields like so: `pdfcpu form list` 

If you are dealing with form data only you can use pdfcpu to export and then refill the form.

**Comment by hhrutter** [comment]
closed due to inactivity
